ext/standard/php_dir.h

```diff
@@ -17,12 +17,59 @@
 #ifndef PHP_DIR_H
 #define PHP_DIR_H
 
-/* directory functions */
-PHP_MINIT_FUNCTION(dir);
-PHP_RINIT_FUNCTION(dir);
+#ifdef HAVE_GLOB
+#ifndef PHP_WIN32
+#include <glob.h>
+#else
+#include "win32/glob.h"
+#endif
+#endif
+
+#ifdef HAVE_GLOB
+
+#ifndef GLOB_BRACE
+# define GLOB_BRACE 0
```

Comment:
Oh wait, I've just realized something. Now, all these `GLOB_*` C constants will be defined... As a consequence, all the relevant PHP constants will be as well. In order to avoid this issue, the `#ifdef` checks in the stub file should also check whether the C constant values are not 0. According to  https://sites.uclouvain.be/SystInfo/usr/include/glob.h.html, they are all have a different value, so this should work.